test/com/google/javascript/jscomp/parsing/NewParserTest.java

```diff
@@ -538,10 +532,9 @@ public void testJSDocAttachment5() {
    * propagate to following code due to {@link JSDocInfo} aggregation.
    */
   public void testJSDocAttachment6() throws Exception {
```

Comment:
For every test you've added in the new file, delete it from here if it was only testing bad jsdocs